backend/src/packages/auth/auth.controller.ts

```diff
@@ -33,6 +36,17 @@ class AuthController extends BaseController {
           }>,
         ),
     });
+
```

Comment:
Add comments for swagger documentation as it's done for user service

Reply:
done. Please tell me if something should be changed